src/bsp/src/stm32/include/USB.h

```diff
@@ -0,0 +1,30 @@
+#ifndef HIVE_MIND_USB_H
+#define HIVE_MIND_USB_H
+
+#include "bsp/ICRC.h"
+#include "bsp/IUSB.h"
+#include <BaseTask.h>
+#include <FreeRTOS.h>
+#include <Mutex.h>
+#include <array>
+#include <c-common/circular_buff.h>
+#include <cstdint>
+#include <logger/ILogger.h>
```

Comment:
Maybe cleanup unused headers (or move them to the .cpp if they are needed there)